src/collection-view.js

```diff
@@ -249,11 +209,11 @@ var CollectionView = Backbone.View.extend({
 
       // since append moves elements that are already in the DOM,
       // appending the elements will effectively reorder them
-      this._appendReorderedChildren(elsToReorder);
```

Comment:
My mistake, I actually thought "CompositeView is gone so we don't need it", but it's only _deprecated_.